website/docs/language/modules/develop/providers.mdx

```diff
@@ -182,10 +182,10 @@ The keys of the `providers` map are provider configuration names as expected by
 the child module, and the values are the names of corresponding configurations
 in the _current_ module.
 
-Once the `providers` argument is used in a `module` block, it overrides all of
-the default inheritance behavior, so it is necessary to enumerate mappings
-for _all_ of the required providers. This is to avoid confusion and surprises
-that may result when mixing both implicit and explicit provider passing.
+If the unaliased `provider` name is not present in the providers map, 
+it will still be inherited in the same way as implicit provider inheritance.
+`providers` argument used in `module` block will not override the default 
+inheritance behavior.
```

Comment:
```suggestion
Terraform implicitly passes the un-aliased provider name to child objects, even if it is excluded from the `providers` map. As a result, child objects still inherit the provider name according to the provider inheritance. Using the `providers` argument in a `module` block does not override the default inheritance behavior.
```
Is this clearer and still mean what you intended?

Reply:
@jbardin removing that para won't help. User should be aware what happens for the provider block which doesn't have alias. Resources for unalias provider block still gets created same way even if we do not pass it explicitly in module hence I think what @trujillo-adam is clearer.